distribution/docs/src/main/resources/_contents/_endpoints/cometd-endpoint-contents.adoc

```diff
@@ -0,0 +1,756 @@
+
+In order to retrieve notifications that were persisted or may have occurred since the previous session a client simply must publish an empty json message, `{}` to `/${ddf-branding-lowercase}/notifications`.
+This will return all existing notifications to the user.
+
+====== Activity Events Channel
+
+To receive all activity updates, follow the instructions at <<asynchronous_search_subscribing, Subscribing to Notifications>> and subscribe to `/ddf/activities/**`
+
+Activity update messages follow a specific format when they are published to the activities channel.
+These messages contain a data map that encapsulates the activity information.
+
+.CometD Activity Format
+[cols="1m,3,1,3,3" options="header"]
+|===
+|Map Key
+|Description
+|Value Type
+|Possible Values
+|Example Values
+
+|category
+|Category of the activity
+|String
+|Any String
+|"Product Retrieval"
+
+|id
+|ID that uniquely identifies the activity that sent out the update. Not required
+to be unique per update.
+|String
+|Any String
+|"b72ccdf6-8ca7-4f53-a0f6-b0ad264393b0"
+
+|message
+|User-readable message that explains the current activity status
+|String
+|Any String
+|"Resource retrieval downloading."
+
+|operations
+|Map of operations that can be performed on this activity
+|JSON Map
+|A map of operations that can be used by clients to communicate back to the server on the same channel.
+
+a|
+
+``"operations" : {
+	"download" : "http://example.com/product"
+}`` +
+If the value is a URL, the client should invoke the URL as a result of the user invoking the activity operation.
+
+If the value is not a URL, the client should send a message back to the server on the same topic with the operation name.
+
+Note: the ${ddf-branding} UI will interpret several values with special icons: +
+
+* `download` +
+* `retry` +
+* `cancel` +
+* `pause` +
+* `remove` +
+* `resume` +
+
+|progress
+|Percentage value of activity completion
+|String
+|Integer between 0 - 100 followed by a %
+|"45%"
+
+|status
+|Enumerated value that displays the current state of the activity
+|String
+a|
+* `STARTED` +
+* `RUNNING` +
+* `FINISHED` +
+* `STOPPED` +
+* `PAUSED` +
+* `FAILED`
+|`RUNNING`
+
+|timestamp
+|Time that the activity update was sent
+|String
+|Positive long value (seconds
+since unix epoch)
+|`1403801920875`
+
+|title
+|User-readable title for the activity update
+|String
+|Any String
+|"Download Complete"
+
+|subject
+|User who started the activity
+|String
+|Any String
+|"admin"
+
+|bytes
+|Number of bytes the activity consumed (upload or download)
+|Integer
+|Positive Integer
+|11
+
+|session
+|The session ID of the user/subject
+|String
+|Any String
+|"6q2lfmmwh9dwv1c4gevboyp297"
+
+|Custom Value
+|Additional keys can be inserted by the component sending the activity notification
+|Any JSON Type
+|Any JSON
+|"{}"
+
+|===
+
+.Example: Activity update with custom 'bytes' field
```

Comment:
We need to review what is actually being sent in the activities and update the table and example accordingly.